### ISSUE TITLE
What if ML server runs on different port than 8000

### ISSUE THREAD

**Issue by tomchrcoder** [marklogic/marklogic-data-hub#404]
I'm trying to use data-hub (version 1.1.2) for a PoC. Due restrictions in the dev data center, our ML server runs on different ports. Instead of using port 8000 we had to use port 30000. Where can I change this so that when data-hub (MarkLogicAuthenticationManager.authenticate) tries to connect, it uses port 30000 instead? Any help greatly apreciated.

**Comment by paxtonhare** [comment]
@tomchrcoder you should be able to set

```
mlAppServicesPort=30000
```

in your gradle.properties file.

I did grep the code and found 1 instance of 8000 being hard coded though. So give that a try and reply back if it works. Otherwise that 1 rogue instance may be an issue.

**Comment by tomchrcoder** [comment]
@paxtonhare thank you for swift reply.

The hint using -mlAppServicesPort didn't work as it seems HubConfig class is not considering this property. However I found a command line parameter --mlRestPort=30000 which solved that particular problem

Now I'm getting stopped at ManageClient.getXml at line 114 where it tries to connect to port 8002.

For this I found in ManageClient  newManageClient the port 8002 hard coded. Is there a way to patch this without re compiling? As I do not have a Java development environment it would be very difficult for me.
